docs/architecture/2023-10-17_autoupdate_v3.md

```diff
@@ -0,0 +1,76 @@
+# Launcher Autoupdate Process, Version 3
+
+## Authors
+
+- Rebecca Mahany-Horton ([@RebeccaMahany](https://github.com/RebeccaMahany))
+
+## Status
+
+Accepted: October 2023
+
+Changes rolled out to nightly channel in October 2023. Rollout to beta and
+stable is still TBD.
+
+Supersedes: [Launcher Auto Update Process Version 2](2019-09-05_autoupdate_v2.md)
+
+## Context
+
+Our current autoupdate process has several areas we'd like to improve upon.
+
+First, updates are currently stored on the end user device under the timestamp
+when they were downloaded. This does not allow launcher to know what version
+the update corresponds to.
+
+Second, release channels are currently implemented by copying the most recent
+version of an autoupdatable binary to `<binary>-<channel>.tar.gz` and publishing
+this copy to Notary. We would like a more flexible implementation that does not
+require storing a binary twice.
+
+Third, we would like the location of updates to be configurable.
+
+See [Begin work on Autoupdate: The Next Generation](https://github.com/kolide/launcher/issues/954)
+for more details.
+
+## Decision
+
+We created a new update directory that lives, by default, in the root
+directory, but its location is configurable via flag.
+
```

Comment:
Probably need a sentence or paragraph here saying something like:

"We will use the go-tuf implementation of TUF to track our releases and what versions are in each channel."